docs/manual/common/concepts/InternalAndExternalCommunication.md

```diff
@@ -1,26 +1,40 @@
 </ul></ul>  
 
-* Publish with the [[Message Broker API|MessageBrokerAPI]] --- for at-least-once semantics. If a new instance starts publishing information, its messages are added to events previously emitted. If a new instance subscribes to a topic, they will receive all events, past, present, and future (as long as they are subscribed). Lagom implements the API to work with [[Kafka|KafkaServer]], but you could also implement your own. 
+* Databases and other persistent storage can be seen as another way that nodes of a service communicate.For microservices that use persistent entities, Lagom encourages [[event streaming|ES_CQRS]], which also takes advantage of asynchronous communication and provides guarantees via an event log.
+
+This diagram illustrates each of these types of inter- and intra-service communication in a Lagom system distributed across three servers. In the example, the Order service publishes to one or more Kafka topics, while the User service subscribes to consume the information. The User service communicates with other User service instances (cluster members) using Akka remoting. The Shipping service and User service exchange information by streaming it in service calls. [[ServiceCommunication.png]] 
+
+## Communication with parties outside of a microservices system
+
+Lagom promotes use of asynchronous communication without preventing use of synchronous communication where necessary. For such extra-service communication, third parties can obtain data asychronously from Lagom services that publish to the Broker API and enjoy the at-least-once guarantee. Lagom services also expose an API for third parties to exchange data synchronously. This is usually mapped to HTTP. Lagom Service APIs also support streaming data to external clients by means of websockets. See [[ServiceDescriptors]] for more information. 
 
-> Note: For microservices that use persistent entities, Lagom encourages [[event streaming|ES_CQRS]], which also takes advantage of asynchronous communication and provides guarantees via an event log.
+Interaction with the outside world could mean clients that use the services over the internet, such as browsers, mobile apps or IoT devices. While using standard HTTP or WebSockets, typically such clients do not communicate directly with individual services. Usually, a network boundary acts as a perimeter, and a well-controlled communication point acts as the intermediary between the outside world and the inside world. In Lagom, this communication point is service gateway. Envision your microservices system as a Medieval town with a wall around it and one gate offers the only way in or out. Communication inside the walls is free and direct, but communication to or from the outside world must come through the service gateway, as illustrated in the following graphic. [[ExtraSystemCommunication.png]]
```

Comment:
"In Lagom, this communication point is service gateway." sounds like it's missing a word or two... "is known as a service gateway" or "is called a service gateway" or just "is a service gateway"?